unit-test/background/privacy-configuration-tests.js

```diff
@@ -0,0 +1,63 @@
+require('../background/mock-browser-api')
+
+const tdsStorageStub = require('./../helpers/tds.es6')
+const tdsStorage = require('../../shared/js/background/storage/tds.es6')
+
+const Site = require('../../shared/js/background/classes/site.es6')
+const utils = require('../../shared/js/background/utils.es6')
+const browserWrapper = require('../../shared/js/background/wrapper.es6')
+
+const contentScriptUtils = require('../../shared/js/content-scope/utils.js')
+
+const testSets = require('./reference-tests/privacy-configuration/tests.json')
+const configs = {
+    // FILE_MAP ./reference-tests/privacy-configuration/config*_reference.json
```

Comment:
this will get replaced with

```js
'config1': require('./reference-tests/privacy-configuration/config1_reference.json'),
'config2': require('./reference-tests/privacy-configuration/config2_reference.json'),
'config3': require('./reference-tests/privacy-configuration/config3_reference.json'),
```